packages/chonky/src/components/file-list/ListEntry.tsx

```diff
@@ -38,37 +39,46 @@ export const ListEntry: React.FC<FileEntryProps> = React.memo(
         const ChonkyIcon = useContext(ChonkyIconContext);
         const fileEntryHtmlProps = useFileEntryHtmlProps(file);
         return (
-            <div className={classes.listFileEntry} {...fileEntryHtmlProps}>
-                <div className={commonClasses.focusIndicator}></div>
+            <div id="chonkyListFileEntry" className={classes.listFileEntry} {...fileEntryHtmlProps}>
+                <div id="chonkyFocusIndicator" className={commonClasses.focusIndicator}></div>
                 <div
+                    id="chonkySelectionIndicator"
                     className={c([
                         commonClasses.selectionIndicator,
                         classes.listFileEntrySelection,
                     ])}
                 ></div>
-                <div className={classes.listFileEntryIcon}>
+                <div id={file?.isDir ? "chonkyListDirEntryIcon" : "chonkyListFileEntryIcon"} className={classes.listFileEntryIcon}>
                     <ChonkyIcon
                         icon={dndIconName ?? entryState.icon}
                         spin={dndIconName ? false : entryState.iconSpin}
                         fixedWidth={true}
                     />
                 </div>
                 <div
+                    id="chonkyListFileEntryName"
                     className={classes.listFileEntryName}
                     title={file ? file.name : undefined}
                 >
                     <FileEntryName file={file} />
                 </div>
-                <div className={classes.listFileEntryProperty}>
+                <div id="chonkyFileModDate" className={classes.listFileEntryProperty}>
                     {file ? (
-                        fileModDateString ?? <span>—</span>
+                        fileModDateString ?? <span></span>
                     ) : (
                         <TextPlaceholder minLength={5} maxLength={15} />
                     )}
                 </div>
-                <div className={classes.listFileEntryProperty}>
+                <div id="chonkyFileSizeList" className={classes.listFileEntryProperty}>
                     {file ? (
-                        fileSizeString ?? <span>—</span>
+                        fileSizeString ?? <span></span>
+                    ) : (
+                        <TextPlaceholder minLength={10} maxLength={20} />
+                    )}
+                </div>
+                <div id="chonkyUploadedByEmail" className={classes.listFileEntryProperty}>
+                    {file?.uploadedByEmail && !file?.isDir ? (
```

Comment:
Yes, the file can be a dir.

Reply:
i tried locally without the file is dir check and it didn't seem to look any different

Reply:
Did you try both the grid view and list view? Without the isDir check I would expect to see in the list view mode the text "Uploaded by: " for folders.

Reply:
No, in my environment, uploaded-by doesn't show to folders

Reply:
Don't know what happens in the local env, but `ListEntry` is used both for folders and files.

Reply:
I'm sorry, you are correct, my mistake. That field won't show for folders because we don't have the "uploadedBy" filed on the folder serializer. I've removed the `isDir` check.